invenio_rdm_records/services/access/service.py

```diff
@@ -754,6 +754,13 @@ def create_guest_access_request(self, identity, token, expand=False, uow=None):
             uow=uow,
         )
 
+    def _translate_expiration_date(self, value):
+        """Translate secret_link_expiration from ui dropdown value."""
```

Comment:
We should not assume in the backend that there is a UI dropdown component in the frontend, this is REST API :)